lib/plugins/shs.js

```diff
@@ -37,30 +37,31 @@ module.exports = {
 
   /**
    * @param {any} api
-   * @param {Config} config
+   * @param {Config & {multiserverShs?: {cap?: string; seed?: Buffer}}} config
    */
   init (api, config) {
     /** @type {number | undefined} */
     let timeoutHandshake
-    if (u.isNumber(config.timers?.handshake)) {
-      timeoutHandshake = config.timers?.handshake
+    if (u.isNumber(config.global.timers?.handshake)) {
+      timeoutHandshake = config.global.timers?.handshake
     }
     if (!timeoutHandshake) {
-      timeoutHandshake = config.timers ? 15e3 : 5e3
+      timeoutHandshake = config.global.timers ? 15e3 : 5e3
     }
     // set all timeouts to one setting, needed in the tests.
-    if (config.timeout) {
-      timeoutHandshake = config.timeout
+    if (config.global.timeout) {
+      timeoutHandshake = config.global.timeout
     }
 
-    const shsCap = (config.caps && config.caps.shs) ?? config.appKey
+    const shsCap = config.multiserverShs?.cap ?? config.global.caps?.shs ?? config.global.appKey
     if (!shsCap) {
       throw new Error('secret-stack/plugins/shs must have caps.shs configured')
     }
+    const seed = config.multiserverShs?.seed ?? config.global.seed
 
     const shs = Shs({
-      keys: config.keys && toSodiumKeys(config.keys),
-      seed: config.seed,
+      keys: config.global.keys && toSodiumKeys(config.global.keys),
```

Comment:
Are these "networkKeys"?

Different pr I think, but is that a change to consider (not baking a feedId into the stack?)

Reply:
This is keys as in ssb-keys.

Reply:
I know what they are - ed25519 keys.
I'm noticing you're about to make a breaking change and "keys" is prrrrretty vague for 2024.

Reply:
What's the suggestion? And do breaking changes have to be bundled together? In my opinion, it's much easier to keep up with new major versions if they require only one breaking change tweak to downstream code, not several changes.